prov/verbs/src/verbs_cm.c

```diff
@@ -123,23 +123,41 @@ vrb_msg_ep_prepare_cm_data(const void *param, size_t param_size,
 
 static inline void
 vrb_ep_prepare_rdma_cm_param(struct rdma_conn_param *conn_param,
-				struct vrb_cm_data_hdr *cm_hdr,
-				size_t cm_hdr_data_size)
+				void *priv_data, size_t priv_data_size)
 {
-	conn_param->private_data = cm_hdr;
-	conn_param->private_data_len = (uint8_t)cm_hdr_data_size;
+	conn_param->private_data = priv_data;
+	conn_param->private_data_len = (uint8_t)priv_data_size;
 	conn_param->responder_resources = RDMA_MAX_RESP_RES;
 	conn_param->initiator_depth = RDMA_MAX_INIT_DEPTH;
 	conn_param->flow_control = 1;
 	conn_param->rnr_retry_count = 7;
 }
 
+static void
+vrb_msg_ep_prepare_rdma_cm_hdr(void *priv_data,
+				const struct rdma_cm_id *id)
+{
+	struct vrb_rdma_cm_hdr *rdma_cm_hdr = priv_data;
+
+	rdma_cm_hdr->ip_version = 0;
+	rdma_cm_hdr->port = 0;
```

Comment:
This is the source port.  Lower 16-bits of src_addr->sib_sid.

Reply:
Thanks. I created a new patch to fix the port and the ip_version.

Reply:
Can you fix the commit message?  I'm not sure 'based on Sean's review' will be particularly useful when reading the git log.  :)

Reply:
I fixed the commit message. Sorry for the rush.